site/src/Librecores/ProjectRepoBundle/Resources/views/Project/view.html.twig

```diff
@@ -6,18 +6,40 @@
 
 {% block pagepath %}Home &raquo; Projects &raquo; {{ project.displayName }}{% endblock %}
 
+{% block stylesheets %}
+  {{ parent() }}
+  <link rel="stylesheet" href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css">
```

Comment:
Please don't include from CDN but copy to the other JS files in the code base. (This also prevents the problem that "latest" could change and be incompatible with our code.)